src/psyclone/tests/psyir/transformations/intrinsics/matmul2code_trans_test.py

```diff
@@ -549,12 +551,13 @@ def test_validate13():
     vector.children[2] = my_range
     with pytest.raises(TransformationError) as excinfo:
         trans.validate(matmul)
-    assert ("Transformation Error: To use matmul2code_trans on matmul, "
-            "each Range index of the argument 'y' must be a full range "
-            "but found non full range at position 2." in str(excinfo.value))
+    assert (f"Transformation Error: To use {trans.name} on matmul, "
```

Comment:
No f-string...

Reply:
Addressed in c4d60bc3ef52a6d7b2853dd5b3c3b01ce82996dc.